src/test/scala/scala/tools/refactoring/tests/implementations/MarkOccurrencesTest.scala

```diff
@@ -439,4 +438,74 @@ class MarkOccurrencesTest extends TestHelper {
       def doit = for (#(x) <- Seq(#(1), #(2))) yield x
     }
     """)
+
+  @Test
+  def trivialObject() = markOccurrences("""
+    object DasDing/*<-cursor-3*/
```

Comment:
considering German words in code :), in Polish we have 'wi-hajst-er' (written together, hyphens for readability) for Ding